docs/04-operating-scs/08-backups/user_data_backups.md

```diff
@@ -0,0 +1,479 @@
+---
+title: User Data Backups
+type:
```

Comment:
Could we write "Guide" here or are only standardizes labels allowed?

```suggestion
type: Guide
```

Reply:
I looked at other docs pages and this header does not seem mandatory/necessary at all. I replaced it with a single Markdown headline instead.